tests/filecheck/mlir-conversion/with-mlir/dialects/memref/memref_ops_custom.mlir

```diff
@@ -0,0 +1,17 @@
+// RUN: xdsl-opt %s | xdsl-opt | mlir-opt --allow-unregistered-dialect | filecheck %s
+
+%v0 = "test.op"() : () -> (i32)
+%i0 = "test.op"() : () -> (index)
+%i1 = "test.op"() : () -> (index)
+%m = "test.op"() : () -> (memref<2x3xi32>)
+memref.store %v0, %m[%i0, %i1] : memref<2x3xi32>
+%v1 = "memref.load"(%m, %i0, %i1) {"nontemporal" = false} : (memref<2x3xi32>, index, index) -> (i32)
+
+// CHECK:      module {
```

Comment:
I'm checking what do we need to have `builtin.module` instead of `module`.
In the meantime we can merge it!